Src/ILGPU.Tests/ExchangeBufferOperations.cs

```diff
@@ -0,0 +1,204 @@
+﻿using ILGPU.Runtime;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ILGPU.Tests
+{
+    public abstract class ExchangeBufferOperations : TestBase
+    {
+
+        private const int Length = 32;
+
+        protected ExchangeBufferOperations(ITestOutputHelper output, TestContext testContext)
+            : base(output, testContext)
+        { }
+
+        internal static void CopyKernel(Index1 index, ArrayView<long> data)
+        {
+            data[index] -= 5;
+        }
+
+        [Theory]
+        [InlineData(10)]
+        [InlineData(int.MaxValue)]
+        [InlineData(int.MinValue)]
+        [KernelMethod(nameof(CopyKernel))]
+        public void Copy(long constant)
+        {
+            using var exchangeBuffer = Accelerator.AllocateExchangeBuffer<long>(Length);
+            for (int i = 0; i < Length; i++)
+            {
+                exchangeBuffer[new Index1(i)] = constant;
+            }
+            exchangeBuffer.CopyToAccelerator();
+
+            var expected = new long[Length];
+            for (int i = 0; i < Length; i++)
+            {
```

Comment:
Singe line statements `inside` for and `if`  are not encapsulated by `{}` in the ILGPU code base 🔢

Reply:
I will fix this now.